readthedocs/starterguides/userguides/Orchestration_UserGuide.md

```diff
@@ -0,0 +1,118 @@
+        * 'Source Bucket Name': Name of the Source cloud bucket from which objects will be migrated
+    d) Once the fields are filled, click on 'ok' button to initiate the workflow execution
+    e) This will take you to the 'instances' page where the status can be tracked for the instance execution
+    f) Once the status is 'success', user can navigate to the 'Dataflow' and check the migration and also check 'Resource->Buckets' to          check if the objects has been migrated to the destination cloud bucket
+
+![migration bucket](migration_bucket.PNG?raw=true "migration bucket")
+
+### Customized workflow
+If user writes their own Mistral workflows and corresponding actions they can follow following steps to orchestrate it through OpenSDS dashboard. All these steps need to be executed from the OpenSDS ssh console.
+
+#### Steps to follow:
+    a) Copy the custom workflow into the '/opt/stackstorm/packs/opensds/actions/workflows/' directory of stackstorm/stackstorm:latest docker container. Ex: docker cp custom_wf.yaml 58d22927c38f:/opt/stackstorm/packs/opensds/actions/workflows/
+    b) Copy all corresponding actions and their runner scripts into '/opt/stackstorm/packs/opensds/actions/'.
+    c) Go into the bash of StackStorm container (Ex: docker exec -it 85bea886a8c5 /bin/bash)
+    d) Run 'st2ctl reload --register-all'
+    e) Run 'st2 action list|grep opensds' to check if the custom workflow is loaded successfully
+    f) Go the CURL/Postman login and execute /v1beta/{tenant_id}/orchestration/services POST API with required inputs. Check the openapi-specs for details. Ex: Request body for registering service:
+    {
+        "name": "custom workflow",
+        "description": "Create and Resize a volume",
+        "tenant_id": "94bhajo8988",
+        "user_id": "558057",
+        "input": "",
+        "constraint": "",
+        "group": "Custom",
+        "workflows": [
+            {
+                "definition_source": "opensds.custom-wf",
+                "wfe_type": "st2"
+            }
+        ]
+    }
+
+Please note that the wfe_type should be 'st2' as OpenSDS supports only 'StackStorm' as Orchestration Manager currently. You can specify any existing group like 'Provisioning' or 'Migration' or create a new group altogether.
+
+Once these steps are done, user can go into the OpenSDS dashboard and create and execute an instance.
+
+### Checking created instacnes
+Click on the 'instacnes' button on 'Services' page to get the list of all instances of the services being created and its status.
+
+## Known issues
+### Multiple service registration in a loop may fail sometime
+Workaround: If service registration is done through CURL/Postman from a script, put a delay of 60 secs between every service registration
+
+### Others:
+Please refer to https://github.com/opensds/orchestration/blob/master/docs/INSTALL.md for some of the known issues while installting some dependent components.
+
+## References
```

Comment:
You may add reference to Developer Guide here

Reply:
Have added as TODO. Once Developer guide is merged, will add link